lib/msal-react/package.json

```diff
@@ -29,6 +29,9 @@
     "test:coverage": "tsdx test .*.spec.* --coverage",
     "lint": "cd ../../ && npm run lint:react",
     "lint:fix": "npm run lint -- -- --fix",
+    "doc": "npm run doc:generate && npm run doc:deploy",
+    "doc:generate": "typedoc src/index.ts",
+    "doc:deploy": "gh-pages -d ref -a -e ref/msal-react",
```

Comment:
Have you tried deploying?

Reply:
https://azuread.github.io/microsoft-authentication-library-for-js/ref/msal-react/

Reply:
Was getting some sort of permission error when I tried to do it from my machine. May need some help.

Reply:
You can also add this branch to the typedoc workflow file to get it to deploy these typedocs automatically.

Reply:
Cool. That worked!